### ISSUE TITLE
Arte reply broken ("No items found")

### ISSUE THREAD

**Issue by Glandos** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#296]
### Describe the bug
The ARTE replay is broken. I've tested both FR and DE

### To Reproduce
1. Open extension and go to Replay TV
2. Go to International > Arte

### Expected behavior
List of categories should appears.

### Actuel behavior
The error popup shows "No items found".

### Your environment
 - [ ] Android
 - [ ] iOS
 - [X] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [ ] Windows
 - [ ] Other

 - Operating system version/name: Ubuntu 18.04 LTS
 - Kodi version: 18.5
 - Catch-up TV & More version: 0.2.20
 - Installation method: From repository

### Debug log
```
2020-04-28 18:54:51.292 T:139864346986240   FATAL: [Catch-up-TV-&-More.support] No items found
                                            Traceback (most recent call last):
                                              File "/home/xbmc/.kodi/addons/script.module.codequick/lib/codequick/support.py", line 295, in run_callback
                                                redirect = parent_ins._process_results(results)
                                              File "/home/xbmc/.kodi/addons/script.module.codequick/lib/codequick/route.py", line 95, in _process_results
                                                raw_listitems = validate_listitems(raw_listitems)
                                              File "/home/xbmc/.kodi/addons/script.module.codequick/lib/codequick/route.py", line 48, in validate_listitems
                                                raise RuntimeError("No items found")
                                            RuntimeError: No items found
```

**Comment by wwark** [comment]
Hi,
I reproduced on my side.
There is just a few change so i fixed it https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/commit/2c22d7a2169be28673c52125931818bedcd115cf

The fix is available on the last beta version 0.2.21 beta 03.

Regards,

**Comment by wwark** [comment]
Ticket closed.
